api/swagger/server/restapi/resource/swagger.yml

```diff
@@ -1031,6 +1070,22 @@ definitions:
       address:
         description: Event caller.
         type: string
+  Get_addresses : 
+    type : object
+    description : rpc get_addresses call
+    properties : 
+      pendingBalances : 
```

Comment:
This returned value feel weird. And the more it will grow, the more it will be.

I would have use an array of map with the key being an address.
```json
  {
    "XXX_ADDR_XXX":
    {
      "balance": { "pending": 0, "final": 0},
      "XXX_other attribute_XXX": XXX,
    },
    "XXX_ADDR2_XXX":
    {
      "balance": { "pending": 0, "final": 0},
      "XXX_other attribute_XXX": XXX,
    },
  }
```